### PR TITLE
update meteor example according to config best practices.

### PR DESCRIPTION
**Pull request by mikedanese**
related to #11361

### CODE REVIEW

File: examples/meteor/meteor-service.json [hunk 1]
```diff
@@ -14,7 +14,6 @@
     "selector": {
       "name": "meteor"
     },
-    "sessionAffinity": "ClientIP",
-    "type": "LoadBalancer"
```
Comment (caesarxuchao) [inline examples/meteor/meteor-service.json]: why do you remove the loadbalancer?

Comment (mikedanese) [inline examples/meteor/meteor-service.json]: accident, I thought it was a default for a second. I have readded it.

**Comment by caesarxuchao** [comment]
LGTM. Thanks!

**Comment by nikhiljindal** [comment]
LGTM, thanks!

**Comment by davidopp** [comment]
#11460 will fix the shippable failure here, but it's not merged yet

**Comment by mikedanese** [comment]
#1487

**Pull request merged**